### DISCUSSION TITLE
[Feature] Add payment methods in Chinese Mainland

### DISCUSSION THREAD

**Discussion by tim-gong** [immich-app/immich#13888] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It is strongly recommended to add payment methods in Chinese Mainland.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
We have no direct control over this; please email billing@futo.org with your request to show your interest.
